postgres/datadog_checks/postgres/postgres.py

```diff
@@ -368,6 +369,8 @@ def _collect_stats(self, instance_tags):
             metric_scope.append(FUNCTION_METRICS)
         if self._config.collect_count_metrics:
             metric_scope.append(self.metrics_cache.get_count_metrics())
+        if self._config.collect_slru_metrics and self.version >= V13:
```

Comment:
Any reason to make this metric opt-in? I think in general, it's better to make metrics like these opt-out unless there are performance concerns (are there?)

Reply:
Is this the exhaustive list of all SLRU names? https://pgpedia.info/p/pg_stat_slru.html

Reply:
I'm more than happy to remove the opt-in parameter. I've assumed looking at the rest of the code that it was prefered for new metrics to be opt-in but given that this one is a simple stats table, the performance impact is negligible. 

If you're talking about the SLRU cache names, they are harcoded in the code and haven't changed since they were introduced  https://github.com/postgres/postgres/blob/e351f85418313e97c203c73181757a007dfda6d0/src/include/utils/pgstat_internal.h#L271-L280